spack.yaml

```diff
@@ -4,23 +4,54 @@
 # configuration settings.
 spack:
   specs:
-  # TODO: Replace the MODEL and VERSION.
-  # The root SBD for the model and overall version of the deployment:
-  # - MODEL@git.VERSION
+    - access-am3@git.2025.06.000
   packages:
-    # TODO: Specify versions and variants of dependencies where required
-    # Specification of dependency versions and variants goes here.
-    # CI/CD requires that the first element of the require is only a version:
-    # openmpi:
-    #   require:
-    #     - '@4.0.2'
+    # Direct dependencies
+    um:
+      require:
+        - '@13.1'
+        - model="vn13p1-am"
+        - jules_sources="2025.08.0"
+        - um_sources="2025.08.0"
+
+    # Indirect dependencies
+    eccodes:
+      require:
+         - '@2.34.0'
+
+    netcdf-c:
+      require:
+        - '@4.9.2'
+
+    netcdf-fortran:
+      require:
+        - '@4.5.2'
+
+    fcm:
+      require:
+        - '@2021.05.0'
+        # TODO: Generalize this Gadi-specific variant for spack.yaml
+        - 'site=nci-gadi'
+
+    gcom:
+      require:
+        - '@7.9'
```

Comment:
Just noting `rAM3` is using `gcom/8.3`. No idea if this is important, but perhaps it is useful just to be consistent?

https://github.com/ACCESS-NRI/ACCESS-RAM3/blob/main/spack.yaml#L34